tests/test_ui_textual_components.py

```
@@ -0,0 +1,641 @@
+
+    @pytest.mark.asyncio
+    async def test_metrics_row_displays_cards(self, mock_state):
+        """Test MetricsRow displays all metric cards."""
+        from textual.app import App
+
+        class TestApp(App):
+            def compose(self):
+                yield MetricsRow(state=mock_state)
+
+        app = TestApp()
+        async with app.run_test() as pilot:
```

Ruff (0.14.14)

Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)